CHANGELOG.md

```diff
@@ -1,58 +1,69 @@
+- `Rint`, `Rdouble`, `Rbool` and `Rcplx`: `NA`-aware wrapper classes for scalar elements of R vectors [[#274]](https://github.com/extendr/extendr/pull/274), [[#284]](https://github.com/extendr/extendr/pull/284), [[#301]](https://github.com/extendr/extendr/pull/301), [[#338]](https://github.com/extendr/extendr/pull/338), [[#350]](https://github.com/extendr/extendr/pull/350)
+- `Integers`, `Doubles`, `Strings`, `Logicals` and `Complexes`: `ALTREP` wrappers for R vectors that deref to slices of the above types (`RInt` etc). [[#274]](https://github.com/extendr/extendr/pull/274), [[#284]](https://github.com/extendr/extendr/pull/284), [[#301]](https://github.com/extendr/extendr/pull/301), [[#338]](https://github.com/extendr/extendr/pull/338), [[#350]](https://github.com/extendr/extendr/pull/350)
+- `ExternalPtr`, a wrapper class for creating R objects containing any Rust object. [[#260]](https://github.com/extendr/extendr/pull/260)
+- [**graphics**] Support for R graphics and graphics devices. The `graphics` feature flag is disabled by default. [[#279]](https://github.com/extendr/extendr/pull/279), [[#360]](https://github.com/extendr/extendr/pull/360), [[#373]](https://github.com/extendr/extendr/pull/373), [[#379]](https://github.com/extendr/extendr/pull/379), [[#380]](https://github.com/extendr/extendr/pull/380), [[#389]](https://github.com/extendr/extendr/pull/389)
+- `Deref` implementation for vector types (Rint/Rfloat/Rbool/Rstr/Robj) to appropriately typed Rust slices. [[#327]](https://github.com/extendr/extendr/pull/327)
+- `default` option for `extendr`-annotated functions, allowing them to have default values, e.g. `fn fred(#[default="NULL"] x: Option<i32>) { }`. [[#334]](https://github.com/extendr/extendr/pull/334)
+- `r_name` option for `extendr`-annotated functions, allowing the generated R function to have a different name. e.g.
+    ```rust
+    #[extendr(
+        use_try_from = true,
+        r_name = "test.rename.rlike",
+        mod_name = "test_rename_mymod"
+    )]
+    fn test_rename() { }
+    ```
+    [[#335]](https://github.com/extendr/extendr/pull/335)
+- `serde::Serialize` implementation for R types. [[#305]](https://github.com/extendr/extendr/pull/305), [[#355]](https://github.com/extendr/extendr/pull/355)
+- `Rany` type and the `as_any` conversion method. [[#320]](https://github.com/extendr/extendr/pull/320)
+- `std::fmt::Debug` implementation for wrapper types. [[#345]](https://github.com/extendr/extendr/pull/345)
+- `[[` operator that works with Rust classes. Its behavior is identical to that of the `$` operator. [[#359]](https://github.com/extendr/extendr/pull/359)
+- `Load` and `Save`, traits that, once implemented, provide the ability to load and save R data in the RDS format. These traits are implemented for all `Robj`. [[#363]](https://github.com/extendr/extendr/pull/363)
+- Support for R version 4.2. [[#399]](https://github.com/extendr/extendr/pull/399)
```

Comment:
#399 just tweaked the CI setting. I think we didn't do any notable changes to support R 4.2 (maybe we replaced some API use that was removed on R 4.2?).

Reply:
```suggestion
- Support for R version 4.2. [[#235]](https://github.com/extendr/extendr/issues/235)
```

https://github.com/extendr/extendr/issues/235 seems more appropriate to refer to.

Reply:
I think I'll reference both, then. No reason not to.

Reply:
I'm fine with either, but Is it a common practice to include changes only on the repository settings? What we publish to crates.io is Rust code and the related docs, so I thought it was out of scope of a changelog. Sorry, I just don't know well about how a Rust crate would usually do.

Reply:
Oh, I see your point. You're right, I'm generally trying to keep this changelog user focused. Which is a bit hard to define, but  the CI is surely out of scope.